.github/workflows/reusable_testing.yml

```diff
@@ -0,0 +1,52 @@
+name: reusable_testing
+
+on:
+  workflow_call:
+    inputs:
+      artifact_name:
+        description: The name of the Tarantool build artifact
+        default: ubuntu-focal
+        required: false
+        type: string
+
+jobs:
+  run_tests:
+    strategy:
+      matrix:
+        metrics-version: [ "0.12.0" ]
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Clone the crud module
+        uses: actions/checkout@v3
+        with:
+          repository: slavakirichenko/crud
+          ref: 'slavakirichenko/add-reusable-testing-workflow'
+
+      - name: Download the Tarantool build artifact
+        uses: actions/download-artifact@v2
+        with:
+          name: ${{ inputs.artifact_name }}
+
+      - name: Install Tarantool
+        # Now we're lucky: all dependencies are already installed. Check package
+        # dependencies when migrating to other OS version.
```

Comment:
What does this mean? Does this workflow check migration?

Reply:
It means that we are lucky with dependencies for now. Pay attention, we use `dpkg` here, not `apt`. If we run this workflow on `ubuntu-22.04`, we may get an error about unresolved dependencies. As far as I know, since version 1.1 `apt` can install local packages via `sudo apt install ./foo-1.2.3.deb`. Probably we can do it here.

Reply:
But I am ok if you leave `dpkg` here.